.clang-format

```diff
@@ -7,8 +7,20 @@ IndentWidth:        2
 UseTab:             Never
 BreakBeforeBraces:  Custom
 BraceWrapping:
-  BeforeCatch: true
-  BeforeElse: true
+  AfterClass:             false
```

Comment:
I think this should be true

Reply:
Nah the example documentation is just wrong. This is the behaviour:

True:
```
class MyClass
{
  // blah
}
```

False:
```
class MyClass {
  // blah
}
```